nixnet_examples/programmatic_database_api.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pprint
+import six
+import time
+
+import nixnet
+from nixnet import constants
+from nixnet import types
+from nixnet.system import system
+
+
+pp = pprint.PrettyPrinter(indent=4)
+
+
+def main():
+    with system.System() as my_system:
+        print(list(my_system.databases))
```

Comment:
Opened #173 about this.

The interesting aspects for this situation
- Can we support integer indexes?  This requires the database list returned by the driver to be a stable order
- Is it ok if we still implement get/del?  Since get is just a function, probably.  del is fine; it works even on mutable sequences (granted, we won't have a fully mutable sequence).